contracts/modules/dispute/DisputeModule.sol

```diff
@@ -411,14 +438,53 @@ contract DisputeModule is
         return $.isWhitelistedArbitrationRelayer[arbitrationPolicy][arbitrationRelayer];
     }
 
-    /// @notice Arbitration policy for a given ipId
+    /// @notice Returns the arbitration policy for a given ipId
     /// @param ipId The ipId
-    /// @return policy The address of the arbitration policy
     function arbitrationPolicies(address ipId) external view returns (address policy) {
         DisputeModuleStorage storage $ = _getDisputeModuleStorage();
         return $.arbitrationPolicies[ipId];
     }
 
+    /// @notice Returns the next arbitration policy for a given ipId
+    /// @param ipId The ipId
+    function nextArbitrationPolicies(address ipId) external view returns (address policy) {
+        DisputeModuleStorage storage $ = _getDisputeModuleStorage();
+        return $.nextArbitrationPolicies[ipId];
```

Comment:
same as above.

Reply:
addressed in https://github.com/storyprotocol/protocol-core-v1/pull/281/commits/7ee122a7561bd398a573ba5bacbc00d5704ac712